x-pack/plugins/endpoint/public/embeddables/resolver/store/middleware.ts

```diff
@@ -78,5 +99,42 @@ export const resolverMiddlewareFactory: MiddlewareFactory = context => {
         }
       }
     }
+    /**
+     * REMOVE: pending resolution of https://github.com/elastic/endpoint-app-team/issues/379
```

Comment:
should remove before merge, can be tracked in the issue itself

Reply:
I thought these might be helpful to the reader/maintainer, but I see they appear a little obtrusive. I'll pull them out.

Reply:
https://github.com/elastic/kibana/pull/64121/commits/fe22f984589afc8852ea4307b650ca1d994df5b9